Makefile.standard_app

```diff
@@ -47,6 +47,14 @@ ifeq ($(ENABLE_SWAP), 1)
     DEFINES += HAVE_SWAP
 endif
 
+#####################################################################
+#                              APP STORAGE                          #
+#####################################################################
+ifeq ($(ENABLE_APP_STORAGE), 1)
```

Comment:
Small nitpick is that this option is not very self explanatory
Maybe something like `ENABLE_OS_MANAGED_APP_STORAGE`, or `ENABLE_UPDATE_RESILIENT_APP_STORAGE` or `ENABLE_PERSISTENT_APP_STORAGE`